content/rules/bridges.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Bridges
+---
+
+# Bridges
+
+Problem: Not all the people or organizations you follow may have migrated from Twitter to open social systems like Mastodon, or they moved to a different decentralized network like Bluesky or Nostr which do not use ActivityPub as their federation protocol.
+
+One option is to follow such accounts on other networks using a bridge.
+
+There are a few issues here:
+
+- While some people may intend to follow accounts from another network through a bridge and understand the limitations, these accounts end up federated and searchable by anyone else in the instance.
+- These bridges may not respect the original author's privacy, or require them to opt-out of federation, republishing without the original author's permission or consent.
+- There are a variety of backend issues when these bridged accounts are followed, unfollowed, replied to, and so on, and their implementations don’t respond properly because they're overloaded or incorrectly implemented.
```

Comment:
```suggestion
- There are a variety of backend issues when these bridged accounts are followed, unfollowed, replied to, and so on. For example, often the bridge implementations does not respond properly because it's overloaded or incorrectly implemented.
```